.pre-commit-config.yaml

```diff
@@ -0,0 +1,46 @@
+# for when we add pre-push hooks, so users only need to run pre-commit install
+default_install_hook_types: [pre-commit, pre-push]
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: debug-statements
+    -   id: name-tests-test
+        args:
+          - "--pytest-test-first"
+    -   id: requirements-txt-fixer
+- repo: local
+  hooks:
+  - id: isort
+    name: isort
+    entry: isort
+    # assumes isort is installed in local virtual environment as dev dependency
+    language: system
+    types: [python]
+  - id: flake8
+    name: flake8
+    entry: flake8
+    # assumes flake8 is installed in local virtual environment as dev dependency
+    language: system
```

Comment:
Let's add `args: ["--profile", "black"]` here as well

Reply:
Hi, I have:
```toml
[tool.isort]
profile = "black"
```

In the `pyproject.toml`, so `isort` picks this up. I think I would prefer to keep the configuration for the pre-commit/linting tooling in one place as much as possible (outside of flake8, since it doesn't support `pyproject.toml` yet), and so think we should avoid setting the profile in both the `pyproject.toml`, and the `.pre-commit-config.yaml` file explicitly. 

Do you still want me to add the `args` for isort as described in your comment? 
Thanks

Reply:
ok. no need to add the `args` then.